src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityProperties.cs

```diff
@@ -94,6 +117,9 @@ public bool AccessibilityHighlightable
             }
         }
 
+        /// <summary>
+        /// Gets or sets whether the view is animated for accessibility or not.
+        /// </summary>
```

Comment:
You can add: 
`///<remarks>
/// For views, which intend to block notification of theirs position or size change to AT-SPI2, this value should be set as true.
/// Otherwise it is set to false by default and the object position or size change is notified to AT-SPI2. 
///</remarks>`